bc-backend/src/main/java/org/jboss/da/bc/impl/ProjectHiearchyCreator.java

```diff
@@ -246,15 +250,20 @@ private void findExistingBuildConfiguration(ProjectDetail project) {
     }
 
     /**
-     * Sets internallyBuilt
+     * Looks up built versions for the ProjectDetail. Sets internallyBuilt, availableVersions.
      */
-    private void checkInternallyBuilt(ProjectDetail project) {
+    private void getBuiltVersions(ProjectDetail project) {
         try {
+
+            List<String> versionsBuilt = versionFinder.lookupBuiltVersions(project.getGav())
+                    .getAvailableVersions();
```

Comment:
You can actually reuse the ``VersionLookupResult`` object result from the lookup method also for bestMatchVersion. How it is now it works too, but it is finding the versions twice and it can take some time to find the versions in Indy.